netbox/templates/login.html

```diff
@@ -46,7 +46,21 @@ <h2 class="text-center mb-4">{% trans "Log In" %}</h2>
               <input type="hidden" name="next" value="{{ request.POST.next }}" />
             {% endif %}
 
-            {% render_form form %}
+            <div class="form-group">
+            <label for="id_username" style="font-weight: bold; margin-bottom: 5px;">{{ form.username.label }}</label>
```

Comment:
We should avoid using inline CSS wherever possible. Bootstrap provides plenty of convenient [utility classes](https://getbootstrap.com/docs/5.3/utilities/text/) for this purpose.